modules/common/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/SpecHelpers.scala

```diff
@@ -63,11 +72,16 @@ object SpecHelpers {
   }"""
 
   /** Builds an Iglu client from the above Iglu configuration. */
-  val client: IgluCirceClient[Id] = IgluCirceClient
-    .parseDefault[Id](igluConfig)
+  val client: IgluCirceClient[IO] = IgluCirceClient
+    .parseDefault[IO](igluConfig)
     .value
+    .unsafeRunSync()
```

Comment:
I don't like that but changing this would require *a lot* of refactoring in the tests. Instead I would prefer us to make it possible to create an `IgluCirceClient` manually without having to parse a config file, so that we don't get a `IO[ ]`.